book/src/clauses/well_known_traits.md

```diff
@@ -0,0 +1,57 @@
+# Well known traits
+
+Not all traits can be encoded in Rust's type system: special traits
+like `Sized`, `Drop` or `Unsize` need additional compiler support in order to
+function properly. To address this, chalk introduces a notion of `WellKnownTrait`s:
+a subset of rustc's trait lang items that need special handling in trait system logic.
+
+As an example, consider the following two aspects of `Sized` logic:
+  1) In order to prove that a struct implements `Sized`, we need to prove 
+     that the last field of that struct is `Sized`.  
+  2) Structs need all of their fields, except, maybe, the last one to be `Sized`.
```

Comment:
While these two points are valid, the flow is a bit weird. I would say something like
> As an example, consider the logic for `Sized` in regards to structs: A struct can have at most one `!Sized` field, and it must be the last. And the last field isn't `Sized`, then neither is the struct itself.